programs/dual-farming/src/lib.rs

```diff
@@ -852,8 +849,6 @@ pub struct Pool {
     /// [] because short size, fixed account size, and ease of use on
     /// client due to auto generated account size property
     pub funders: [Pubkey; 4], // 32 * 4 = 128
-    //the version of the pool
-    pub version: PoolVersion, // 1
```

Comment:
We need to update the `space` when initialize pool PDA

Reply:
space = 8 + 494 // discriminator + content + buffer
   
Currently space is 500 (buffer also), so we can keep this number even